airbyte-integrations/bases/base-normalization/normalization/transform_catalog/table_name_registry.py

```diff
@@ -352,7 +352,7 @@ def get_nested_hashed_table_name(name_transformer: DestinationNameTransformer, s
     """
     parent = "_".join(json_path[:-1])
     max_length = name_transformer.get_name_max_length()
-    json_path_hash = hash_json_path([schema] + json_path)
+    json_path_hash = name_transformer.normalize_table_name(hash_json_path([schema] + json_path), False, False)
```

Comment:
@ChristopheDuong i changed a unit test and the hash part doesn't return upper case, added this to do this.

Reply:
I don't think it's needed here, we have to normalize for capitalization at the end of building the table name (the complete name is not decided yet at this point)